libs/env/src/env.ts

```diff
@@ -761,6 +761,11 @@ const envDefinitions: Record<string, EnvDef<boolean | string | number | string[]
     parser: boolParser,
     desc: "Enable network request and responses logs. Errors are always logged",
   },
+  CRYPTO_ASSET_SEARCH_KEYS: {
+    def: ["ticker", "name", "keywords"],
+    parser: stringArrayParser,
+    desc: "Fuse search in `ticker`, `name`, `keywords` values in cryptoassets list",
```

Comment:
I'd change this description, we already know in the def attributes what is the content

Something more like : "attributes that fuse search will use to find a currency according to user input"

Minor